spec/factories/listings.rb

```diff
@@ -1,7 +1,6 @@
 FactoryBot.define do
   factory :listing do
     location
```

Comment:
i'll check out how to make sure listing db/model don't require location

Reply:
It's already optional in the db, so that's good. I'll update the factory accordingly